src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/model/CcdCase.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.model;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class CcdCase {
```

Comment:
I would name it BulkScanCase rather than a generic CcdCase

Reply:
done